src/test/scala/is/hail/annotations/UnsafeSuite.scala

```diff
@@ -203,19 +203,24 @@ class UnsafeSuite extends SparkSuite {
   @Test def testRegion() {
     val buff = Region()
 
-    buff.appendLong(124L)
-    buff.appendByte(2)
-    buff.appendByte(1)
-    buff.appendByte(4)
-    buff.appendInt(1234567)
-    buff.appendDouble(1.1)
-
-    assert(buff.loadLong(0) == 124L)
-    assert(buff.loadByte(8) == 2)
-    assert(buff.loadByte(9) == 1)
-    assert(buff.loadByte(10) == 4)
-    assert(buff.loadInt(12) == 1234567)
-    assert(buff.loadDouble(16) == 1.1)
+    val o1 = buff.appendLong(124L)
+    val o2 = buff.appendByte(2)
+    val o3 = buff.appendByte(1)
+    val o4 = buff.appendByte(4)
+    val o5 = buff.appendInt(1234567)
+    val o6 = buff.appendDouble(1.1)
+
+    assert(buff.loadLong(o1) == 124L)
+    assert(buff.loadByte(o2) == 2)
+    assert(buff.loadByte(o3) == 1)
+    assert(buff.loadByte(o4) == 4)
+    assert(buff.loadInt(o5) == 1234567)
+    assert(buff.loadDouble(o6) == 1.1)
+    assert(o2 - o1 == 8)
```

Comment:
You're assuming that everything gets allocated out of the first block, which is
relying on internal implementation details.  Probably not going to change, but
a little questionable.